notebooks/00_quick_start/tfidf_covid.ipynb

```diff
@@ -0,0 +1,1235 @@
+{
```

Comment:
<p>in other notebooks we are avoiding to use <code>import *</code>		instead we import the particular functions that are loaded</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1088/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Fixed.